cmd/snap-confine/snap-confine.c

```diff
@@ -523,10 +531,25 @@ int main(int argc, char **argv)
 	// Now that we've dropped and regained SYS_ADMIN, we can load the
 	// seccomp profiles.
 	if (sc_apply_seccomp_profile_for_security_tag(invocation.security_tag)) {
+		sc_explain_li("Applied seccomp profile: %s",
+			   invocation.security_tag);
+                sc_explain_start_section("");
+		sc_explain("source: /var/lib/snapd/seccomp/bpf/%s.src",
```

Comment:
there still too much punctuation around, these need be taken care by some other kind of helper

Reply:
I added a new "sc_explain_kv()" helper to format key/value paris. However to go all the way there either needs to be a "sc_explain_li_kv()" to express listitems of key/value pairs. Or this we could add a format options struct or something else that I have not thought of yet. ideas welcome.

Reply:
I pushed the sc_explain_li_kv() helper, feedback welcome, not great but open for ideas